src/SupportMessage/_tests_/SupportMessage.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import 'jest-styled-components'
+
+import { SupportMessage, SupportMessageProps } from '../SupportMessage'
+
+const supportMessageProps: SupportMessageProps = {
+  description: 'Type info support message',
+  type: 'info',
+}
+
+describe('SupportMessage component', () => {
```

Comment:
No need to have it inside the `_tests_` folder here, I don't think it brings much in this context, it made sense for snapshot as it create quite a few files